openfold/data/feature_pipeline.py

```diff
@@ -119,7 +120,7 @@ def np_example_to_features(
             dtype=torch.float32,
         )
 
-    return {k: v for k, v in features.items()}
+    return {k: v for k, v in features.items()},gt_features
```

Comment:
This is going to fail for monomer I think because gt_features is not defined unless it goes in the else statement

Reply:
Also try to leave spaces between comma separated things

Reply:
> This is going to fail for monomer I think because gt_features is not defined unless it goes in the else statement

But is_multimer is always going to be True even the input structure is a monomer as in here: https://github.com/aqlaboratory/openfold/blob/3aaf0ca89f0dce388d87bdfb44d5653258e9c801/openfold/data/data_modules.py#L511